### ISSUE TITLE
Apollo-link-http has wrong graphql dependency

### ISSUE THREAD

**Issue by VladShcherbin** [apollographql/apollo-link#438]
When installing apollo-link-http@1.3.2 and graphql 0.12, it gives warning:

```bash
warning " > apollo-link-http@1.3.2" has incorrect peer dependency "graphql@^0.11.0".
```

Version 1.3.2 was published with [graphql version 0.11](https://github.com/apollographql/apollo-link/blob/9da1e33b9044ac146d9a91cd33f9cc221c52a713/packages/apollo-link-http/package.json#L48). The dependency was updated later. This [changelog](https://github.com/apollographql/apollo-link/blob/master/packages/apollo-link-http/CHANGELOG.md) is wrong.

**Comment by jbjonesjr** [comment]
![image](https://user-images.githubusercontent.com/533967/35603199-296f32e0-060a-11e8-9f65-409cd61518b7.png)

**Comment by farzd** [comment]
getting warning as well

**Comment by joenot443** [comment]
Is there a workaround for this? My app is currently not building due to this as I'm getting the telltale 
`GraphQL error: Schema must be an instance of GraphQLSchema. Also ensure that there are not multiple versions of GraphQL installed in your node_modules directory.`.

**Comment by evans** [comment]
This should be fixed in [1.3.3](https://github.com/apollographql/apollo-link/blob/master/packages/apollo-link-http/package.json#L48)!
